pytest_xvfb.py

```diff
@@ -130,8 +132,13 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
-    if config.getoption('--no-xvfb') or not xvfb_available():
+    if config.getoption('--no-xvfb') or not sys.platform.startswith('linux'):
         config.xvfb = None
+    elif (not xvfb_available() and sys.platform.startswith('linux') and
+          os.environ.get('DISPLAY') is not None):
+        config.xvfb = None
+        warnings.warn(
+            'pytest-xvfb could not find Xvfb. You can install it to avoid windows from opening.')
```

Comment:
@The-Compiler after giving it some thought, I agree that the pytest_configure function could be much simpler, so I refactored it with your ideas. One thing to note is that the warnings.warn function is picky about breaks in strings so line 139 will break PEP8's line length guideline.

Here is an example of the current warning output:
```
 UserWarning: pytest-xvfb could not find Xvfb. You can install it to avoid windows from opening.
  'pytest-xvfb could not find Xvfb. You can install it to avoid windows from opening.')
```

Here is what the output looks like with a line break between the two sentences in the warning.
```
UserWarning: pytest-xvfb could not find Xvfb. You can install it to avoid windows from opening.
  'pytest-xvfb could not find Xvfb. '
```

I'm not sure why the warning is printed twice, and the documentation for warnings seems to be sparse.